src/scss/lr7-classic-theme/_toolbars.scss

```diff
@@ -266,6 +262,65 @@
 	}
 }
 
+// Management Toolbar Skin
+
+.management-toolbar-default {
+	background-color: $toolbar-default-bg;
+	border-bottom: 1px solid $toolbar-bottom-border;
+	padding: 0 24px;
+
+	.btn {
+		background-color: transparent;
+		border-width: 0;
+		padding: 0;
+	}
+
+	.dropdown-menu {
+		margin-top: $toolbar-dropdown-margin-top;
+	}
+
+	.toolbar-group, .toolbar-group-right, .toolbar-group-expand {
+		height: $management-toolbar-height;
+
+		a, .btn, .btn-primary {
+			color: $toolbar-default-link-color;
+
+			&:hover {
+				color: $toolbar-default-link-hover-color;
+				text-decoration: none;
+			}
+		}
+
+		.active.btn {
+			background-color: $management-toolbar-btn-active-bg;
+		}
+
+		&:first-child, &:first-child .toolbar-group-content, &:first-child .toolbar-group-content > a{
+			padding-left: 0;
+		}
+		&:last-child, &:last-child .toolbar-group-content, &:last-child .toolbar-group-content > a{
+			padding-right: 0;
+		}
+	}
+}
+
+@media screen and (max-width: 767px) {
```

Comment:
We are using the mobile first approach. Styles should be declared for mobile devices and desktop styles should be declared in @media screen and (min-width: 768px) {}.

@media screen and (max-width: 767px) {} should only be used for specific cases where the styles are specific to mobile devices that support media queries and shouldn't be parsed ie8.